tests/test_persisted_channel_properties.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for validating persisted channel properties for different data types."""
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+
+
+class TestPersistedChannelPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for
+    persisted channel properties for different data types.
+    """
+
+    @pytest.fixture(scope="class")
+    def voltage_tester_channel(self):
+        """Gets the voltage tester channel from the persisted channel in the system."""
+        system = nidaqmx.system.System.local()
+        voltage_tester_channel = system.global_channels["VoltageTesterChannel"]
+        return voltage_tester_channel
```

Comment:
Add a `persisted_channel` property to conftest.py like `persisted_scale` in https://github.com/ni/nidaqmx-python/pull/236, plus the suggested skip logic.

Reply:
Added a fixture for the persisted channel in the conftest.py